tools/docs/config.ts

```diff
@@ -131,6 +131,14 @@ function genTable(obj: [string, string][], type: string, def: any): string {
         }
         el[1] = `\n\`\`\`json\n${stringify(el[1], { indent: 2 })}\n\`\`\`\n`;
       }
+      // print enum values for `allowedValues` in JSON notation
+      if (el[0] === 'allowedValues') {
```

Comment:
Can't this condition be rolled into that of line 124? The body of the if-block is identical.